src/deluge/gui/context_menu/clip_settings/clip_settings.h

```diff
@@ -14,8 +14,11 @@ class ClipSettingsMenu final : public ContextMenu {
 	bool setupAndCheckAvailability();
 	bool canSeeViewUnderneath() override { return true; }
 	bool acceptCurrentOption() override; // If returns false, will cause UI to exit
+	void focusRegained() override;
 
 	Clip* clip = nullptr;
+	int32_t clipX = -1;
```

Comment:
There's already tracker variables for the last grid presses, no need to create more

Reply:
> There's already tracker variables for the last grid presses, no need to create more

I did it this way because I was thinking maybe we'd want to add the clip settings menu to the row view and blink the mute pad.

if we did that, then passing the X and Y instead of referencing session view's variables would make more sense